ansible/README.md

```diff
@@ -14,38 +14,34 @@ and the other shortcut commands it provides.
   Originally this way because its contents are secret, it would now be preferable
   for this file to live encrypted in `environments/` or `environmental_defaults/`.
 
-
 ## Local vagrant setup
 
-**Note** This section is quite old, and the latest person to revamp this file
-mostly just left this section untouched.
-If you want to use this, ask around to see if anyone can tell you whether these
-steps currently work or not before going down too much of a rabbit hole
-trying to follow these docs.
-
-Ensure you have downloaded Vagrant and virtual box
-
-### Multi-machine cluster setup
-
-Then start vagrant:
+Ensure you have downloaded Vagrant and virtual box then start vagrant:
 
 ```
 $ vagrant up
 ```
 
-If you run into issues starting vagrant, see the troubleshooting section at the bottom.
+This will setup 4 VMs:
+* 192.168.33.14 (control)
+* 192.168.33.15 (app1)
+* 192.168.33.16 (db1)
+* 192.168.33.17 (proxy1)
+
+See `environments/development` the configuration of this testing environment.
 
-The `./scripts/reset-vms` command can be run at any time, possibly with a subset of the
```

Comment:
What's the equivalent way to do this now that this PR has been merged? I think it's important to be able to easily reset vagrant machines in order to test deployment from a clean state.

Reply:
```
vagrant destroy [vm]
vagratn up
```

Reply:
To add more detail - the main thing that script was doing was setting up the auth but that's now done during Vagrant provisioning.